src/components/CustomerQuota/ItemsSelection/Item.tsx

```diff
@@ -22,6 +22,24 @@ export const Item: FunctionComponent<{
   const { getProduct } = useContext(ProductContext);
   const identifiers = getProduct(cartItem.category)?.identifiers || [];
 
+  let newIdentifiers = identifiers;
+  const newCartItem = cartItem;
+
+  // these identifiers should only be shown if this redemption is chargeable
+  // so we will filter them out if otherwise
+  if (
+    identifiers.length > 0 &&
+    cartItem.category.includes("lost") &&
+    cartItem.descriptionAlert !== "*chargeable"
+  ) {
+    newIdentifiers = identifiers.filter(
+      (identifier) => identifier.label != "Payment receipt number"
+    );
+    newCartItem.identifierInputs = newCartItem.identifierInputs.filter(
+      (identifier) => identifier.label != "Payment receipt number"
+    );
+  }
```

Comment:
I'm conscious about this block being super specific to a campaign which is not supposed to be the design of the mobile app. So I'd like to know: did we discuss other alternatives, and why did we choose this solution?

If this is the sole/best solution, perhaps we can make it clearer what this chunk is meant for? E.g. wrap this up in a condition for "if this campaign is TT", or extracting this logic out into a helper named specifically for TT. In that case we at least know which campaign caused us to need this logic, and it'll be easier to remove this logic once it's no longer necessary.

Reply:
currently, the mobile app doesnt conditionally render any identifiers, hence we were discussing on how we can best cater for this case, without making too much changes. one way we thought of was conditionally excluding specific identifiers (for this case, the receipt field). this will allow us to add all related additional identifiers in policies.

another possible solution was to conditionally include the receipt field when it's chargeable, but felt like perhaps including part of the policies in code might not be the way moving forward. 

i think we can look into extracting this logic for better clarity, but what do you think of the current solution?😛
(cc @ilhaoni)

Reply:
>i think we can look into extracting this logic for better clarity, but what do you think of the current solution?😛

Actually what I am saying is if we have no other alternatives, we might want to extract this out into an independent block, so that it's easier to do cleanup next time too. e.g. extracting out into a function, and doing a `if` around it (perhaps checking for the campaign name or some kind of campaign identifier), with some comment above to explain the context for which this was added (which will hint at when we can remove it too).

Reply:
good point! i've extracted the entire logic and added a check for tt campaign as well, lemme know if this looks ok!